packages/e2e/test/blockfrost/getAsset.test.ts

```diff
@@ -0,0 +1,30 @@
+import * as envalid from 'envalid';
+import { Cardano } from '@cardano-sdk/core';
+import { assetProviderFactory } from '../../src';
+import { logger } from '@cardano-sdk/util-dev';
+
+// Verify environment.
+export const env = envalid.cleanEnv(process.env, {
+  ASSET_PROVIDER: envalid.str(),
+  ASSET_PROVIDER_PARAMS: envalid.json({ default: {} })
+});
+
+describe('blockfrostAssetProvider', () => {
```

Comment:
Please add a test that asserts presence of `tokenMetadata`

Reply:
@mkazlauskas I checked every asset, but unfortunately, the `blockfrost` `preprod` API doesn't return any asset with the field `metadata`, which we use to populate `tokenMedatadata`.

There is a mocked response test in [BlockfrostAssetProvider.test.ts](https://github.com/input-output-hk/cardano-js-sdk/blob/bfd5a7749b47ef01ac39a344957aa9203f788614/packages/cardano-services/test/Asset/BlockfrostAssetProvider/BlockfrostAssetProvider.test.ts)  which I think covers this case.